src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectOfStyles = {};
+  const stylesArray = sourceString.split(';');
+
+  for (let style of stylesArray) {
+    if (style.trim().length < 3) {
+      continue;
+    }
+
+    style = style.split(':').map((s) => s.trim());
+    objectOfStyles[style[0]] = style[1];
```

Comment:
Before assigning `style[1]` to `objectOfStyles[style[0]]`, ensure that both `style[0]` and `style[1]` are not undefined. This will help avoid potential runtime errors if the input string is not well-formed.